src/jit/gentree.cpp

```diff
@@ -7817,6 +7817,504 @@ GenTreePtr GenTree::GetChild(unsigned childNum)
     }
 }
 
+GenTreeOperandIterator::GenTreeOperandIterator()
+    : m_node(nullptr)
+    , m_operand(nullptr)
+    , m_argList(nullptr)
+    , m_multiRegArg(nullptr)
+    , m_expandMultiRegArgs(false)
+    , m_state(-1)
+{
+}
+
```

Comment:
To be clear, I do agree that we should have one helper for this. From what I've seen, there's very little code outside of the various IR dumpers that uses `NumChildren()`/`GetChild()` and this code tends to perform in-order iteration rather than random access. The dumpers may be a bit less tractable, however, as their access patterns are more complex.